src/Sulu/Bundle/PageBundle/Resources/js/index.js

```diff
@@ -44,7 +44,7 @@ initializer.addUpdateConfigHook('sulu_page', (config: Object, initialized: boole
     );
 
     if (config.versioning) {
-        fieldRegistry.add('page_settings_versions', PageSettingsVersions);
+        fieldRegistry.add('page_settings_versions', SettingsVersions);
```

Comment:
You've changed it back to `page_settings_versions`, right? Renaming it to `settings_versions` was for me, but we can also keep it like this. But then I would also not rename the `PageSettingsVersions` components. Either way is fine for me, but I would keep it consistent in this place.

Reply:
Oops that was by mistake, I would go with `settings_versions`. Otherwise it is kinda confusing that we use `page_settings_versions` in the `article-template'